ml-proto/spec/kernel.ml

```diff
@@ -82,9 +82,9 @@ and expr' =
   | Block of expr list                      (* execute in sequence *)
   | Loop of expr                            (* loop header *)
   | Break of var * expr option              (* break to n-th surrounding label *)
-  | Br_if of var * expr option * expr       (* conditional break *)
+  | Break_if of var * expr option * expr    (* conditional break *)
+  | Break_table of var list * expr option * expr  (* indexed break *)
```

Comment:
This is more of a pre-existing nit: but they're not breaks; breaks only branch forward and these branch forwards and backwards.  How about expanding to "br" to "Branch"?  That way noone will be superficially confused by lack of "Continue" :)